### ISSUE TITLE
Mod not loading after multiple game restarts

### ISSUE THREAD

**Issue by maxymillian** [klyte45/DynamicTextProps#36]
Ive restarted my game multiple times now, yet the WTS mod still doesnt fully load, and has never fully loaded for me.
A window appears which reads:

> THIS IS NOT AN ERROR!!!!!!!!
> Seems the mod "Write The Signs" was not completely loaded. Restart your game to make it be full loaded!
> THIS IS NOT AN ERROR!!!!!!!!

![obraz](https://user-images.githubusercontent.com/88718871/128876005-ffeeae0a-1664-4ee1-a423-cf801c7f6e78.png)

**Comment by threevolve** [comment]
I’m having the same issue.

**Comment by threevolve** [comment]
I unsubscribed a bunch of mods I dumbly had disabled but left installed, and it seems to have solved this problem. For me at least.

**Comment by isaac-prog** [comment]
> I unsubscribed a bunch of mods I dumbly had disabled but left installed, and it seems to have solved this problem. For me at least.

do u know which mod it conflicts with?

**Comment by klyte45** [comment]
I'll close this issue, but please @threevolve if possible could you share which mods have you disabled?

Anyway, this alert is fake news. It's already working despite of this alert. I looking for a solution to it.

**Comment by threevolve** [comment]
> I'll close this issue, but please @threevolve if possible could you share which mods have you disabled?
> 
> Anyway, this alert is fake news. It's already working despite of this alert. I looking for a solution to it.

Actually it’s still not working. The error message no longer appears, but I still haven’t been able to get WTS to do anything, and it doesn’t seem to be saving local settings, only global settings persist.

**Comment by klyte45** [comment]
Hmmm, well, reopening it so

I'm planning a bunch of minor fixes update until the end of the year. Until there, I recommend to just ignore these messages...

**Comment by klyte45** [comment]
@threevolve
I forgot to ask you, but can you provide your log after trying loading WTS? (See Report-a-Bug helper if don't know how to do it)

**Comment by threevolve** [comment]
Yes no problem, next time I boot up the game I’ll grab that for you

**Comment by threevolve** [comment]
> @threevolve I forgot to ask you, but can you provide your log after trying loading WTS? (See Report-a-Bug helper if don't know how to do it)

I have attached my last output log :)
[output_log.txt](https://github.com/klyte45/WriteTheSigns/files/7377598/output_log.txt)

**Comment by klyte45** [comment]
This log seems to not load a city 😅
Can you check it?

**Comment by threevolve** [comment]
Hmm, which file do you need? Maybe I put the wrong one

**Comment by klyte45** [comment]
No, it's this the file, but seems you haven't loaded a city in the session this file was generated

**Comment by threevolve** [comment]
> No, it's this the file, but seems you haven't loaded a city in the session this file was generated

Alrighty, how about this one from last night
[output_log.txt](https://github.com/klyte45/WriteTheSigns/files/7384569/output_log.txt)
ne

**Comment by klyte45** [comment]
Are you using some kind of UI replacement mod? (a skin or whatever)

**Comment by threevolve** [comment]
I am using unified ui if that counts. Other than that no.

**Comment by klyte45** [comment]
Okay...

I read in the log that something is making the panel to be destroyed, so it cause the issue with panel not found... I will need to add a log there to make a test with it. I see you use the Address mod too, I will update it later with this log so I will be able to diagnose what is happening.

I'll tell you when this get ready at workshop.

**Comment by klyte45** [comment]
Well, this got very long and I wasn't able to keep with the support last months

The good news is that will have a overhaul of almost all mods after the next DLC release, including the WTS. So wait for news there.
